src/main/java/com/automattic/elasticsearch/statsd/StatsdService.java

```diff
@@ -155,7 +155,8 @@ public void run() {
                                                 false,                              // discoveryStats
                                                 false,                              // ingest
                                                 false,                              // adaptiveSelection
-                                                false                               // scriptCache
+                                                false,                               // scriptCache
+                                                true                   //indexing pressure
```

Comment:
We should introduce a new setting `metrics.statsd.report.indexing_pressure` with default set to `false` so that this does not change the default stats pushed by this plugin.

Reply:
That makes sense :)